src/appleseed-max-impl/appleseedoslplugin/oslshadermetadata.h

```diff
@@ -86,6 +86,7 @@ class MaxParam
     bool            m_has_constant;
     std::string     m_max_label_str;
     int             m_max_param_id;
+    int             m_max_map_param_id;
```

Comment:
What does the word "map" mean in this context?

Reply:
Map is a texture input. Before id of a texture input was calculated based on float or color parameter id, now it's separate variable.

Reply:
Oh ok, makes sense, thanks.